.changeset/rare-stingrays-think.md

```diff
@@ -0,0 +1,5 @@
+---
+vercel: patch
+---
+
+Update dependencies for CLI
```

Comment:
We still need a Version Packages PR to trigger the release again, so just putting some filler in here to trigger that.